freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/kerberosmgmt/model/CleanupRequest.java

```diff
@@ -0,0 +1,37 @@
+package com.sequenceiq.freeipa.api.v1.kerberosmgmt.model;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.sequenceiq.service.api.doc.ModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel("CleanupV1Request")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CleanupRequest {
+
+    @ApiModelProperty(value = ModelDescriptions.ENVIRONMENT_CRN, required = true)
+    @NotNull
+    private String environmentCrn;
+
+    @ApiModelProperty(value = ModelDescriptions.CLUSTER_CRN)
+    private String clusterCrn;
+
+    public String getEnvironmentCrn() {
+        return environmentCrn;
+    }
+
+    public void setEnvironmentCrn(String environmentCrn) {
+        this.environmentCrn = environmentCrn;
+    }
+
+    public String getClusterCrn() {
+        return clusterCrn;
+    }
+
+    public void setClusterCrn(String clusterCrn) {
+        this.clusterCrn = clusterCrn;
+    }
+}
```

Comment:
It looks like this file may not have a new line at the end of it.

Reply:
no. There is no newline. If there was a newline style check would have failed. :-)